### PR TITLE
add documentation for signup methods

### PR DESCRIPTION
**Pull request by jeriox**
I tried using the workflow we talked about for user docs:
- write in German
- put into DeepL and do a raw proof-read
- (not yet) insert original german content into Weblate

### CODE REVIEW

**Comment by jeriox** [comment]
german version below
```
Um verschiedene Anwendungsfälle abzudecken, bietet ephios verschiedene Anmeldeverfahren an. Nachfolgend sind die mitgelieferten Anmeldeverfahren erläutert, es können jedoch auch zusätzliche Anmeldeverfahren als Plugin integriert werden.

Beim Anlegen einer Schicht muss jeweils ein Anmeldeverfahren ausgewählt werden. Dieses steuert, wie sich Benutzer für die Schicht anmelden können und was nach der Anmeldung geschieht. 

Direkte Bestätigung

Schichten mit diesem Anmeldeverfahren bieten dem Nutzer einen Button mit der Aufschrift "Teilnehmen" an. Jeder Benutzer, der sich anmeldet, wird unmittelbar bestätigt und als Teilnehmer der Veranstaltung geführt. 

In den Einstellungen der Schicht können Bedingungen angegeben werden, die ein Benutzer erfüllen muss, um teilnehmen zu können. Dazu gehört ein Mindestalter und eine Anmeldefrist (Datum). Außerdem kann eine minimale und eine maximale Anzahl von Teilnehmenden angegeben werden. Ist die maximale Anzahl von Teilnehmenden erreicht, können sich keine weiteren Benutzer anmelden. Weiterhin können erforderliche Qualifikationen angegeben werden. Benutzer müssen dabei alle genannten Qualifikationen aufweisen, um teilnehmen zu können.

Außerdem kann eingestellt werden, ob Nutzer, die bereits bestätigt sind, selbstständig wieder absagen können und ob individuell abweichende Start- und Endzeiten bei der Anmeldung angegeben werden dürfen.

Anfragen und bestätigen

Schichten mit diesem Anmeldeverfahren bieten dem Nutzer einen Button mit der Aufschrift "Anfragen" an. Jeder Benutzer, der eine Teilnahme anfragt, wird mit seiner Anfrage in der Disposition gelistet. Die für die Veranstaltung verantwortlichen Personen erhalten eine Benachrichtigung über die Anfrage. In der Disposition können diese die Teilnahme dann bestätigen oder ablehnen. 

Es können für dieses Anmeldeverfahren die selben Einstellungen vorgenommen werden wie bei "Direkte Bestätigung."

Bewerbung auf Abschnitte

Dieses Anmeldeverfahren funktioniert grundlegend genau wie "Anfragen und Bestätigen". Zusätzlich können in den Einstellungen der Schicht mehrere Abschnitte (z.B. "UHS", "Fußstreife") definiert werden. Für jeden Abschnitt kann eine Mindest- und Höchstanzahl an Teilnehmenden angegeben werden. Ebenfalls pro Abschnitt können die erforderlichen Qualifikationen angegeben werden. Benutzer müssen alle genannten Qualifikationen aufweisen, um an diesem Abschnitt teilnehmen zu können. Abschnitte mit gemischten Qualifikationen (z.B. RTW mit 1x RS, 1x NFS) müssen entweder als mehrere Abschnitte abgebildet oder mit einem anderen Anmeldeverfahren durchgeführt werden.

Benutzer können beim Anfragen der Teilnahme einen bevorzugten Abschnitt angeben. In den Einstellungen der Schicht kann eingestellt werden, dass diese Angabe verpflichtend ist. Der bevorzugte Abschnitt ist in der Disposition ersichtlich.

Keine Anmeldung (nur Disposition)

Die Möglichkeit zur Anmeldung an dieser Schicht ist für alle Nutzer deaktiviert, optional kann ein Grund hierfür angegeben werden. Nutzer können über die Disposition hinzugefügt werden. Dieses Anmeldeverfahren kann bspw. für Lehrgänge verwendet werden, bei dem die Verantwortlichen einzelne Helfende entsenden und eine Selbstanmeldung nicht vorgesehen ist.

an andere Schicht gekoppelt

Bei diesem Anmeldeverfahren ist eine Anmeldung für Helfende nicht möglich, stattdessen werden die Teilnahmen von einer anderen Schicht gespiegelt. Die Schicht, von der gespiegelt werden soll, wird in den Einstellungen ausgewählt. Dieses Anmeldeverfahren kann bspw. für mehrtägige Lehrgänge verwendet werden, bei denen die Anmeldung über die erste Schicht geregelt wird und die Teilnehmenden automatisch für alle weiteren Tage/Schichten angemeldet werden sollen.



Disposition

Auf der Detailseite eines Events kann für jede Schicht über den Button "Disposition" die Disposition erreicht werden. Dort werden alle Teilnahmen für die Schicht in den zugehörigen Zuständen angezeigt.

- Angefragt: Der Nutzer hat eine Teilnahme für diese Schicht angefragt. Es werden zusätzlich die relevanten Qualifikationen des Nutzers und Zusatzinformationen wie der bevorzugte Abschnitt angezeigt. Befindet sich am Zahnrad ein Ausrufezeichen, wurde bei der Anmeldung eine abweichende Teilnahmezeit oder ein Kommentar angegeben. Diese können über einen Klick auf das Zahnrad eingesehen werden.
- Abgesagt durch verantwortliche Person: die Teilnahme wurde von einer verantwortlichen Person über die Disposition abgelehnt.
- Abgesagt durch helfende Person: die helfende Person hat selbstständig für diese Schicht abgesagt.
- Bestätigt: die Teilnahme wurde durch eine verantwortliche Person bestätigt. Hier können (sofern benötigt) weitere Angaben wie der zugeteilte Abschnitt gemacht werden. Über das Zahnrad kann der Teilnehmer für eine abweichende Zeit eingeteilt werden. 
- Weitere hinzufügen: Über das Suchfeld kann eine Teilnahme für weitere Benutzer angelegt werden, die für diese Schicht bisher keine Anmeldung in ephios getätigt haben. Wird für den eingegebenen Namen kein Benutzer gefunden, bietet ephios die Möglichkeit, einen Platzhalter mit diesem Namen anzulegen. Dieser kann beispielsweise genutzt werden, wenn externe Helfende ohne Account eingeteilt werden sollen.

Der Zustand einer Teilnahme kann durch das Verschieben des Eintrags in das gewünschte Feld geändert werden. Beispielsweise kann eine angefragte Teilnahme bestätigt werden, indem sie in das Feld "Bestätigt" verschoben wird. Die Änderungen müssen anschließend mit "Speichern" bestätigt werden.
```

File: docs/user/planner/signup_methods.rst [hunk 1]
```diff
@@ -0,0 +1,57 @@
+Signup methods
+==============
+
+To cover different use cases, ephios offers different signup methods. In the following, the provided logon procedures are explained, but additional signup methods can also be integrated as a plugin.
```
Comment (felixrindt) [inline docs/user/planner/signup_methods.rst:4]: I think this misses the point of explaining what signup methods actually are. "Logon" is something I'd connect to my login credentials. I think we should motivate why we provide different signup methods.

The next paragraph should maybe come first.

File: docs/user/planner/signup_methods.rst [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+Shifts with this signup methods offer the user a button with the inscription "Participate". Any user who signs up is immediately confirmed and listed as a participant in the shift.
+
+The shift settings can specify conditions that a user must meet in order to participate. These include a minimum age and a registration deadline. In addition, a minimum and a maximum number of participants can be specified. If the maximum number of participants is reached, no further users can register. Furthermore, required qualifications can be specified. Users must have all the specified qualifications to be able to participate.
+
+In addition, it can be set whether users who have already been confirmed can cancel on their own and whether individually deviating start and end times can be specified when registering.
+
+Request and confirm
+^^^^^^^^^^^^^^^^^^^
+
+Shifts with this registration procedure offer the user a button labeled "Requests". Any user requesting to attend will have their request listed in the disposition. The persons responsible for the event will receive notification of the request. In the disposition they can then confirm or reject the participation.
```
Comment (felixrindt) [inline docs/user/planner/signup_methods.rst:22]: I don't think we should introduce yet another term ("registration procedure").

Comment (felixrindt) [inline docs/user/planner/signup_methods.rst:22]:
```suggestion
Shifts with this registration procedure offer the user a button labeled "Request". Any user requesting to attend will have their request listed in the disposition. The persons responsible for the event will receive notification of the request. In the disposition they can then confirm or reject the participation. 
```

File: docs/user/planner/signup_methods.rst [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+The shift settings can specify conditions that a user must meet in order to participate. These include a minimum age and a registration deadline. In addition, a minimum and a maximum number of participants can be specified. If the maximum number of participants is reached, no further users can register. Furthermore, required qualifications can be specified. Users must have all the specified qualifications to be able to participate.
+
+In addition, it can be set whether users who have already been confirmed can cancel on their own and whether individually deviating start and end times can be specified when registering.
+
+Request and confirm
+^^^^^^^^^^^^^^^^^^^
+
+Shifts with this registration procedure offer the user a button labeled "Requests". Any user requesting to attend will have their request listed in the disposition. The persons responsible for the event will receive notification of the request. In the disposition they can then confirm or reject the participation. 
+
+The same settings can be made for this signup method as for "Instant confirmation."
```
Comment (felixrindt) [inline docs/user/planner/signup_methods.rst:24]:
```suggestion
The same settings can be made for this signup method as for "Instant confirmation".
```

File: docs/user/planner/signup_methods.rst [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+No registration (disposition only)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The ability to sign up for this shift is disabled for all users, optionally a reason can be specified. Users can be added via the disposition. This registration procedure can be used e.g. for training courses, where the responsible persons send individual helpers and a self-registration is not intended.
+
+Linked to other shift
+^^^^^^^^^^^^^^^^^^^^^
+
+With this registration method, it is not possible for volunteers to register; instead, the participants are mirrored by another shift. The shift to be mirrored is selected in the settings. This registration procedure can be used, for example, for courses lasting several days, where registration is controlled via the first shift and the participants are automatically registered for all other days/shifts.
```
Comment (felixrindt) [inline docs/user/planner/signup_methods.rst:41]:
```suggestion
With this registration method, it is not possible for volunteers to register; instead, the participants are mirrored from another shift. The shift to mirror is selected in the settings. This registration procedure can be used, for example, for courses lasting several days, where registration is controlled via the first shift and the participants are automatically registered for all other days/shifts.
```

File: docs/user/planner/signup_methods.rst [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+Disposition
+^^^^^^^^^^^
+
+On the detail page of an event, the disposition can be reached for each shift via the button "Disposition". There, all participations for the shift are displayed in the corresponding states.
```
Comment (felixrindt) [inline docs/user/planner/signup_methods.rst:46]:
```suggestion
On the detail page of an event, responsibles of the event can reach the disposition for each shift via the button "Disposition". There, all participations for the shift are displayed in the corresponding states.
```

File: docs/user/planner/signup_methods.rst (hunk 2 above)
Comment (jeriox) [reply docs/user/planner/signup_methods.rst:22]: yeah that is due to DeepL. as stated above, I only checked the english version roughly. DeepL uses a lot of different words for the same german word. This is all Anmeldeverfahren in german, but DeepL uses registration procedure, logon procedure, signup procedure etc. in the same translation run. Same was true for thinks like Zahnrad where i had cog, cogwheel and gear in the translation

**Review by felixrindt — APPROVED** [review]
lgtm if you add stubs for the planned structure

**Pull request merged**